core/src/main/java/org/elasticsearch/rest/action/cat/RestNodesAction.java

```diff
@@ -85,7 +85,7 @@ public RestChannelConsumer doCatRequest(final RestRequest request, final NodeCli
         clusterStateRequest.clear().nodes(true);
         clusterStateRequest.local(request.paramAsBoolean("local", clusterStateRequest.local()));
         clusterStateRequest.masterNodeTimeout(request.paramAsTime("master_timeout", clusterStateRequest.masterNodeTimeout()));
-
+        final boolean fullId = request.paramAsBoolean("full_id", false);
```

Comment:
This isn't where I would expect it to be consumed since it affects the output only, not the request handling.